src/main/java/com/gluonhq/substrate/model/InternalProjectConfiguration.java

```diff
@@ -461,21 +461,26 @@ private void checkGraalVMPermissions(String graalvmHome) {
         if (graalvmHome == null || graalvmHome.isEmpty()) {
             return;
         }
-        Logger.logDebug("Checking execution permissions for " + graalvmHome);
+        String graalvmRoot = graalvmHome.endsWith("Contents/Home") ?
```

Comment:
Checking for `/Contents/Home` is a little bit more correct.

Reply:
Makes sense, fixed